lib/translations/pt-BR.js

```diff
@@ -0,0 +1,56 @@
+
+
+module.exports = {
```

Comment:
Looks reasonable to me to have it as `.json` but I don't see if it is that important